apps/web/src/features/organizations/components/AddAccounts/index.tsx

```diff
@@ -80,12 +82,14 @@ const AddAccounts = () => {
       })
 
       if (result.error) {
-        // TODO: Handle error message
+        // TODO: Add error handler and display more specific error
```

Comment:
If there is time we can add a proper error handler but currently the types are not accurately working.